frontend/src/scenes/insights/trendsLogic.ts

```diff
@@ -305,21 +353,23 @@ export const trendsLogic = kea({
                 people.next
             )
         },
-        [actions.loadMorePeople]: async (_, breakpoint) => {
-            const { people: currPeople, count, action, label, day, breakdown_value, next } = values.people
-            actions.setLoadingMorePeople(true)
-            const people = await api.get(next)
-            actions.setLoadingMorePeople(false)
-            breakpoint()
-            actions.setPeople(
-                [...currPeople, ...people.results[0]?.people],
-                count + people.results[0]?.count,
-                action,
-                label,
-                day,
-                breakdown_value,
-                people.next
-            )
+        loadMorePeople: async ({}, breakpoint) => {
+            if (values.people) {
+                const { people: currPeople, count, action, label, day, breakdown_value, next } = values.people
+                actions.setLoadingMorePeople(true)
+                const people = await api.get(next)
+                actions.setLoadingMorePeople(false)
+                breakpoint()
+                actions.setPeople(
+                    [...currPeople, ...people.results[0]?.people],
+                    count + people.results[0]?.count,
+                    action,
+                    label,
+                    day,
+                    breakdown_value,
+                    people.next
+                )
+            }
```

Comment:
since we're properly type checking now, what should we do if listeners do nothing here? maybe emit an error so we can log it? and show UI error

Reply:
Not sure what you mean - if they do nothing in what sense?

Reply:
yeah ok we can roll with that